.gitignore

```diff
@@ -31,3 +31,6 @@ venv_docker/
 # Generated files
 cache/
 output/
+
+# Ignore editor configs
+.vscode
```

Comment:
I'd argue we want this in the repo to avoid anyone's `.vscode` folders getting added (similar with other editor configs, though don't know those to add now).

Reply:
Ideally everyone should have their own editor's configuration ignored in their git `core.excludesFile` on their machine to prevent them from adding it to any repo they contribute to. I believe a repo's .gitignore file should only have items related to the project or tools required for the project. Looking through this project's .gitignore now though it seems that there are some things that can be removed, such as the sql and ruby stuff that has not been a part of this project for many years. I should clean that up. In the end if a contributor makes a mistake those files should be caught in the PR process.

Reply:
I'll avoid any cleanup to the `.gitignore` in this PR and just revert my change (after deleting the `.vscode` folder, it was just some spelling ignores anyway).